mir_perception/mir_barrier_tape_detection/ros/launch/barrier_tape_detection.launch

```diff
@@ -39,6 +34,49 @@
             # 1 : Single threaded
             # 2->inf : Parallelism level
             concurrency_level: 1
-        </rosparam>
-  </node>
+          </rosparam>
+      </node>
+  </group>
+
+  <group ns="mir_perception/back_camera">
+      <node pkg="mir_barrier_tape_detection" type="barrier_tape_detection_node" name="barrier_tape_detection"  output="screen" respawn="true">
+          <remap from="~input_rgb_image" to="/camera/color/image_rect_color"/>
+          <remap from="~input_pointcloud" to="/camera/depth_registered/points"/>
+          <remap from="~camera_info" to="/camera/color/camera_info"/>
+          <param name="loop_rate" type="int" value="1" />
+          <param name="target_frame" value="map"/>
+          <remap from="~event_in" to="/mir_perception/barrier_tape_detection/event_in"/>
+          <remap from="~output/yellow_barrier_tape_pointcloud" to="/mir_perception/barrier_tape_detection/output/yellow_barrier_tape_pointcloud"/>
+          <rosparam> 
+        color_thresh_min_h: 30
+        color_thresh_min_v: 28
+          </rosparam> 
+      </node>
+
+      <!-- Point cloud to laser scan conversion-->
+      <node pkg="pointcloud_to_laserscan" type="pointcloud_to_laserscan_node" name="pointcloud_to_laserscan">
+          <remap from="cloud_in" to="/mir_perception/barrier_tape_detection/output/yellow_barrier_tape_pointcloud"/>
```

Comment:
this topic needs to be changed too I think. Both seem to be publishing to the same one